packages/notebook/src/widget.ts

```diff
@@ -2652,6 +2653,38 @@ export class Notebook extends StaticNotebook {
     return null;
   }
 
+  /**
+   * Find anchor target by any ID in any of the cell.
+   */
+  async _findLegacy(queryId: string): Promise<Private.IScrollTarget | null> {
```

Comment:
This slipped through:

```suggestion
  private async _findLegacy(queryId: string): Promise<Private.IScrollTarget | null> {
```

Actually the methods above should be private too but that is my mistake of past which I can correct separately.

Reply:
I updated it